python/pyspark/sql/types.py

```diff
@@ -815,6 +815,96 @@ def jsonValue(self) -> Dict[str, Any]:
 
     @classmethod
     def fromJson(cls, json: Dict[str, Any]) -> "StructType":
+        """
+        Constructs :class`StructType` from a schema defined in json format
+
+        Parameters
+        ----------
+        json : dict or a dict-like object e.g. json object
```

Comment:
Clarify what json object means here. Object from what library?

Reply:
Added JSON schema it must adhere to. I can add the library info but here I am saying that "it must be any dict like object for example json object". So, it can be any custom Python class instance as long as it behaves like dict or json